src/components/creditNote/CreditNoteCodeSnippet.tsx

```diff
@@ -0,0 +1,70 @@
+import { serializeCreditNoteInput } from '~/core/serializers'
+import { CreateCreditNoteInput } from '~/generated/graphql'
+import { CodeSnippet } from '~/components/CodeSnippet'
+import { CreditNoteForm } from '~/components/creditNote/types'
+import { envGlobalVar } from '~/core/apolloClient'
+
+const { apiUrl } = envGlobalVar()
+
+interface CreditNoteCodeSnippetProps {
+  loading: boolean
+  invoiceId: string
+  formValues: CreditNoteForm
+}
+
+const getSnippet = (infos: CreateCreditNoteInput) => {
+  return `# Assign a credit note to an invoice
+curl --location --request POST "${apiUrl}/api/v1/credit_notes" \\
+    --header "Authorization: Bearer $YOUR_API_KEY" \\
+    --header 'Content-Type: application/json' \\
+    --data-raw '{
+      "credit_note": {
+        "invoice_id": "${infos.invoiceId || '__LAGO_INVOICE_ID__'}",
+        "reason": "${infos.reason || 'YOUR_REASON'}",
+        "credit_amount_cents": ${infos.creditAmountCents},
+        "refund_amount_cents": ${infos.refundAmountCents},
+        "items: [
+          ${infos.items.reduce((acc, item, i) => {
+            if (i === infos.items.length - 1) {
+              return (
+                acc +
+                `{
+            "fee_id": "${item.feeId}",
+            "amount_cents": ${item.amountCents}
+          }`
+              )
+            }
+            if (i === 0) {
+              return (
+                acc +
+                `{
+            "fee_id": "${item.feeId}",
+            "amount_cents": ${item.amountCents}
+          },
+          `
+              )
+            }
+
+            return (
+              acc +
+              `{
+            "fee_id": "${item.feeId}",
+            "amount_cents": ${item.amountCents}
+          },
+          `
+            )
+          }, '')}
+        ]
+      }'
+# To use the snippet, don’t forget to edit your __YOUR_API_KEY__ and  __EXTERNAL_CUSTOMER_ID__`
```

Comment:
`__EXTERNAL_CUSTOMER_ID__`  seems not used whereas `__LAGO_INVOICE_ID__` and `YOUR_REASON` are not mentioned here